src/Components/HomeContainer/HomeContainer.js

```diff
@@ -10,24 +10,18 @@ import style from './HomeContainer.module.scss'
 const HomeContainer = () => {
   const [ view ] = useContext(ViewContext)
 
-  return ( 
-    <div className={style.HomeContainer}>
+  return (
+    <div className={style.HomeContainer} id="HomeContainer">
       <VideoProvider>
-        { view === 'timer' &&  
-          <CountdownTimer />
-        }
-        { view === 'content-selection' &&
-          <ContentSelection/>
-        }
-        { view === 'content-delivery' && 
-          <ContentDelivery />
-        }
-        { (view === 'mood-rating-1' || view === 'mood-rating-2') && 
+        {view === "timer" && <CountdownTimer />}
+        {view === "content-selection" && <ContentSelection />}
+        {view === "content-delivery" && <ContentDelivery />}
+        {(view === "mood-rating-1" || view === "mood-rating-2") && (
```

Comment:
This is very nitpicky but you shouldn't need that final opening paren on line 19 or the corresponding closing one on 21

Reply:
Fo sho, I think VSCode auto formatted that one! I'm happy to take it out of you'd prefer!